components/products.jsx

```diff
@@ -0,0 +1,26 @@
+import React, { useEffect, useState } from 'react';
+
+function App() {
+  const [cards, setCards] = useState([]);
+
+  useEffect(() => {
+    fetch('https://panda-market-api.vercel.app/api/products')
+      .then(data => {
+        setCards(data);
+      })
+      .catch(error => {
+        console.error('Error fetching data:', error);
+      });
+  }, []);
+
+  return (
+    <div>
+      <ul>
+        
+      </ul>
```

Comment:
이렇게 내용을 표현해보시고 다른 요소나 스타일도 적용하시기 시작하면 될거에요.
물론 그전에 src/App.js에 이 컴포넌트를 import해서 추가해야겠습니다
```suggestion
      <ul>
        {cards.map(card => (
            <div>{card.name}</div>
        ))}
      </ul>
```